### PR TITLE
[Fixed JENKINS-27002] Claim "Stickiness" default value should be configurable

### PR DESCRIPTION
**Pull request by Batmat**
See https://issues.jenkins-ci.org/browse/JENKINS-27002 for associated issue.

### CODE REVIEW

File: src/main/java/hudson/plugins/claim/AbstractClaimBuildAction.java [hunk 1]
```diff
@@ -34,7 +34,7 @@
     private String claimedBy;
     private String assignedBy;
     private Date claimDate;
-    private boolean transientClaim;
+    private boolean transientClaim = !ClaimConfig.get().isStickyByDefault();;
```
Comment (daniel-beck) [inline src/main/java/hudson/plugins/claim/AbstractClaimBuildAction.java]: Extra `;`

Comment (Batmat) [inline src/main/java/hudson/plugins/claim/AbstractClaimBuildAction.java]: Urgh, thanks for the review. I guess I'll force push the fix on this branch.

**Comment by Batmat** [comment]
@daniel-beck done. Thanks again for the review.

**Comment by Batmat** [comment]
Great! Thanks for merging.
Do you have any ETA for the next release?
Thanks a lot

**Pull request merged**